app/controllers/space/agreements_controller.rb

```diff
@@ -1,11 +1,28 @@
 class Space
   class AgreementsController < ApplicationController
     def show
-      authorize(agreement)
+    end
+
+    def create
+      agreement.save
+
+      if agreement.errors.present?
```

Comment:
Unusual. Why not `if agreement.save`?

Reply:
Ah sorry it's because I've been working in a code base with `decent_exposure` and `responders`. In those, the ergonomics tend to push you towards checking for the presence of errors since the `save` call often happens  automatically.